pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -89,6 +90,30 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		return nil, err
 	}
 	if err := utilerrors.NewAggregate(runtime.DecodeList(list.Objects, kapi.Scheme)); err != nil {
+		return nil, kapierror.NewInternalError(err)
+	}
+
+	// one of the items in this list should be the project.  We are going to locate it, remove it from the list, create it separately
+	var projectFromTemplate *projectapi.Project
+	objectsToCreate := &kapi.List{}
+	for i := range list.Objects {
+		if templateProject, ok := list.Objects[i].(*projectapi.Project); ok {
+			projectFromTemplate = templateProject
+
+			if len(list.Objects) > (i + 1) {
+				objectsToCreate.Items = append(objectsToCreate.Items, list.Objects[i+1:]...)
+			}
+			break
+		}
+
+		objectsToCreate.Items = append(objectsToCreate.Items, list.Objects[i])
+	}
+	if projectFromTemplate == nil {
+		return nil, kapierror.NewInvalid("Template", list.Name, fielderrors.ValidationErrorList{fielderrors.NewFieldInvalid("Template", list.Name, "must contain a project resource")})
```

Comment:
Also, this is not "invalid" because there is no action the user can take.  InternalError is more appropriate